### PR TITLE
JDK 11 preparation and jakarta uptake.

### PR DESCRIPTION
**Pull request by Tomas-Kraus**
Signed-off-by: Tomas Kraus <Tomas.Kraus@oracle.com>

### CODE REVIEW

**Comment by Tomas-Kraus** [comment]
TODO: Updata java.activation dependency when jakarta.activation is released.

File: NOTICE.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+which is available at http://www.eclipse.org/org/documents/edl-v10.php. This
+Source Code may also be made available under the following Secondary Licenses
+when the conditions for such availability set forth in the Eclipse Public
+License v. 2.0 are satisfied: GNU General Public License, version 2 with the GNU
+Classpath Exception which is available at
+https://www.gnu.org/software/classpath/license.html.
+
+SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause OR GPL-2.0 WITH
+Classpath-exception-2.0
+
```
[the remark this answers is not in the document]
Comment (Tomas-Kraus) [reply NOTICE.md]: This is a copy from https://www.eclipse.org/projects/tools/documentation.php?id=ee4j.jaxb

File: jaxb-api/pom.xml [hunk 2]
```diff
@@ -144,7 +138,8 @@
                             <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
                         </archive>
                         <instructions>
-                            <Require-Capability>osgi.ee;filter:="(&amp;(osgi.ee=JavaSE)(version>=1.8))"</Require-Capability>
+                            <_noee>true</_noee>
+                            <!--Require-Capability>osgi.ee;filter:="(&amp;(osgi.ee=JavaSE)(version>=1.7))"</Require-Capability-->
```
[the remark this answers is not in the document]
Comment (Tomas-Kraus) [reply jaxb-api/pom.xml]: Removed

File: NOTICE.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tomas-Kraus) [reply NOTICE.md]: Fixed.

File: jaxb-api-test/pom.xml [hunk 3]
```diff
@@ -49,7 +50,10 @@
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <release>9</release>
+                    <release>11</release>
+                    <compilerArgs>
+                        <arg>-verbose</arg>
```
Comment (lukasj) [inline jaxb-api-test/pom.xml]: should go away

Comment (Tomas-Kraus) [reply jaxb-api-test/pom.xml]: Fixed.

**Comment by Tomas-Kraus** [comment]
Fixed. Now `find ./ -type f -exec grep -Hn 'java.net' {} \;` returns no URLs, just some package imports, etc.

**Comment by lukasj** [comment]
If you look at the bottom of generated javadoc page, does what it says make sense to you?

**Comment by Tomas-Kraus** [comment]
Comments to: jaxb-dev@eclipse.org
More information at: https://projects.eclipse.org/projects/ee4j.jaxb

What's wrong on it?

**Comment by Tomas-Kraus** [comment]
Removed the last line - Oracle copyright.

**Comment by lukasj** [comment]
Yup, that was it. I've just had bad feeling about it. Good you spotted it.

**Pull request merged**